src/components/OrderPanel/BookingDatesForm/BookingDatesForm.js

```diff
@@ -337,21 +337,32 @@ const handleMonthClick = (
 // lineItems from this Template's backend for the EstimatedTransactionMaybe
 // In case you add more fields to the form, make sure you add
 // the values here to the orderData object.
-const handleFormSpyChange = (
+
+const calculateLineItems = (
   listingId,
   isOwnListing,
   fetchLineItemsInProgress,
-  onFetchTransactionLineItems
+  onFetchTransactionLineItems,
+  availabilityType
 ) => formValues => {
-  const { startDate, endDate } =
-    formValues.values && formValues.values.bookingDates ? formValues.values.bookingDates : {};
+  const startDate = formValues.values && formValues.values.startDate;
+  const endDate = formValues.values && formValues.values.endDate;
+  const seats = formValues.values && formValues.values.seats;
+
+  const orderData = {
+    bookingStart: startDate,
+    bookingEnd: endDate,
+    ...(availabilityType && availabilityType === 'multipleSeats' && { seats: parseInt(seats, 10) }),
+  };
 
-  if (startDate && endDate && !fetchLineItemsInProgress) {
+  if (
+    startDate &&
+    endDate &&
+    (availabilityType !== 'multipleSeats' || seats) &&
```

Comment:
```suggestion
    (isSeatsInUse && seats > 0)
```
=> if seats are in use there must be seats value available too.